mapoutput.c

```diff
@@ -183,7 +183,7 @@ outputFormatObj *msCreateDefaultOutputFormat( mapObj *map,
   }
 
   else if( strcasecmp(driver,"AGG/PNG") == 0 ) {
-    if(!name) name="png24";
+    if(!name) name="png";
```

Comment:
Changing to png makes this consistent with the output default. png24 was used previously as png was mapped to the now removed GD/PNG driver.